tests/symbol_table_builder_test.py

```diff
@@ -48,10 +48,10 @@
 class SymbolTableBuilderTest(unittest.TestCase):
     def test(self):
         for filename in os.listdir(os.path.realpath(os.path.join(os.path.dirname(__file__),
-                                                                 os.path.join('..', 'models')))):
+                                                                 os.path.join(os.pardir, "models", "neurons")))):
```

Comment:
why not test synapses any longer?

Reply:
The old code did not list all the files recursively, hence the test was not being run. I have updated the code now to recursively run the test for all `.nestml` files.

Reply:
but only ``models/neurons/*.nestml`` and not ``models/synapses/*.nestml``, right? Does it fail on the latter?

Reply:
I have modified the code to include all model files.